neon_llm_fastchat/fastchat.py

```diff
@@ -54,19 +55,26 @@ def convert_role(role):
             role_fastchat = "Assistant"
         return role_fastchat
 
-    def ask(self, message, chat_history):
+    def assemble_prompt(self, message, chat_history):
         prompt = self.system_message
         # Context N messages
         for role, content in chat_history[-self.context_depth:]:
             role_fastchat = self.convert_role(role)
             prompt += f"### {role_fastchat}: {content}\n"
         prompt += f"### Human: {message}\n### Assistant:"
-        
+        return prompt
+
+    def ask(self, message, chat_history):
```

Comment:
pls add type annotations // not clear what this method is doing, so consider either renaming or adding docstring here

Reply:
Done with https://github.com/NeonGeckoCom/neon-llm-fastchat/pull/5/commits/b4ab71a7ac20b2de693a7798cc8e318385c88c39